src/main/java/seedu/address/logic/commands/eventvolunteercommands/EventListVolunteerCommand.java

```diff
@@ -0,0 +1,77 @@
+package seedu.address.logic.commands.eventvolunteercommands;
+
+import static java.util.Objects.requireNonNull;
+
+import java.util.List;
+import java.util.function.Predicate;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.ToStringBuilder;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.Command;
+import seedu.address.logic.commands.CommandResult;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.event.Event;
+import seedu.address.model.volunteer.Volunteer;
+
+/**
+ * Removes a volunteer from an event.
+ */
+public class EventListVolunteerCommand extends Command {
+
+    public static final String COMMAND_WORD = "elistv";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Lists all volunteer assigned to an event. "
+            + "Example: " + COMMAND_WORD + " 1";
+    public static final String MESSAGE_SUCCESS = "Listed all %2$d volunteers FROM EVENT: %1$s";
+    private final Index eventIndex;
+
+    /**
+     * Creates an EventListVolunteerCommand to add the specified {@code Volunteer} to the {@code Event}
+     */
+    public EventListVolunteerCommand(Index eventIndex) {
+        this.eventIndex = eventIndex;
+    }
+
+    @Override
+    public CommandResult execute(Model model) throws CommandException {
+        requireNonNull(model);
+        List<Event> lastShownEventList = model.getFilteredEventList();
+        if (eventIndex.getZeroBased() >= lastShownEventList.size()) {
+            throw new CommandException(Messages.MESSAGE_INVALID_EVENT_DISPLAYED_INDEX);
+        }
+
+        Event event = lastShownEventList.get(eventIndex.getZeroBased());
+        Predicate<Volunteer> predicateShowVolunteers = v
+                -> event.getAssignedVolunteers().stream().anyMatch(y -> y.equals(v.getName()));
```

Comment:
maybe can make assignedVolunteers a set of Volunteers so that when we do the comparison, we will be entirely sure we are getting correct list of volunteers assigned to the event. For eg, there might be 2 Alice Tan in the volunteer list, but only one of them is assigned to a particular event, but this predicate might return both Alice Tan because they check by Name only. I might be wrong tho

Reply:
hmm good point but... aren't names unique? (else vcreate will shout at you)